logstash_notifier/__init__.py

```diff
@@ -148,10 +152,10 @@ def application(include=None):
             if len(user_data) > 0:
                 extra['user_data'] = user_data
 
-        logger.info(
-            '%s %s', headers['eventname'], event_body['processname'],
-            extra=extra,
-        )
+        message = event_data if len(event_data) \
+                             else '%s %s' % (headers['eventname'], event_body['processname'])
```

Comment:
No; events don't have a message body (event_data), so its set to '' higher up and overridden here with here with the '%s %s' code. 

Stdout/Stderr output DO have a body (obviously), so we use that if it's present.

Added that as a comment for clarity.